locations/spiders/alior_bank_pl.py

```diff
@@ -0,0 +1,40 @@
+import json
+
+from scrapy import Spider
+
+from locations.categories import Categories, Extras, apply_category, apply_yes_no
+from locations.dict_parser import DictParser
+from locations.hours import DAYS, OpeningHours
+
+
+class AliorBankPLSpider(Spider):
+    name = "alior_bank_pl"
+    item_attributes = {"brand": "Alior Bank", "brand_wikidata": "Q9148395"}
+    start_urls = ["https://www.aliorbank.pl/en/branches.html"]
+
+    def parse(self, response, **kwargs):
+        data = json.loads(response.xpath("//script[@id='branch']/text()").get())
+        for city, branches in data.items():
+            for branch in branches:
+                item = DictParser.parse(branch)
```

Comment:
I don't think you're actually getting anything from DictParser here

```suggestion
                item = Feature()
```

Reply:
Changed in be7739437d6c702695747e493754fbe4c188524d